.changes/unreleased/Changed-20220122-011524.yaml

```
@@ -0,0 +1,6 @@
+kind: Changed
+body: '#205 Change the default directory name to .changes when the init command is
+  run and also change the .changie.yml config to use the new directory.'
```

No need to include the PR number here, just the custom issue below is enough.
No need to mention the changie project specific change of moving the changes as the end users won't need to know that.
No need to specify "Change the" as the kind is changed. Just start with "Default directory..."
A little picky on these sorts of changes, sorry.

```suggestion
body: 'Default directory name to .changes for `init` command'
```